src/styles/style.css

```diff
@@ -0,0 +1,269 @@
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(9)::before {
+  content: '9';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(10)::before {
+  content: '10';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(11)::before {
+  content: '11';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(12)::before {
+  content: '12';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(13)::before {
+  content: '13';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(14)::before {
+  content: '14';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(15)::before {
+  content: '15';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(16)::before {
+  content: '16';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(17)::before {
+  content: '17';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(18)::before {
+  content: '18';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(19)::before {
+  content: '19';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(20)::before {
+  content: '20';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(21)::before {
+  content: '21';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(22)::before {
+  content: '22';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(23)::before {
+  content: '23';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(24)::before {
+  content: '24';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(25)::before {
+  content: '25';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(26)::before {
+  content: '26';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(27)::before {
+  content: '27';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(28)::before {
+  content: '28';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(29)::before {
+  content: '29';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(30)::before {
+  content: '30';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(31)::before {
+  content: '31';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
```

Comment:
There is a lot of code duplication for the `.calendar__day:nth-child(n)::before` selector, where n ranges from 1 to 31. Instead of repeating the same styles, consider using a loop in your SCSS file to generate these styles. This will make your code more maintainable and less prone to errors.